tests/safety/test_subaru.py

```diff
@@ -61,6 +61,11 @@ class TestSubaruSafetyBase(common.PandaSafetyTest, MeasurementSafetyTest):
 
   INACTIVE_GAS = 1818
 
+  # Safety around steering req bit
+  MIN_VALID_STEERING_FRAMES = 7
+  MAX_INVALID_STEERING_FRAMES = 1
+  MIN_VALID_STEERING_RT_INTERVAL = 144000
+
```

Comment:
might be a good idea to make these couple of tests a new test class, so it's more explicit when adding this safety.

Reply:
made a common test